python/semantic_kernel/functions/kernel_function_decorator.py

```diff
@@ -137,7 +147,15 @@ def _parse_parameter(name: str, param: Any, default: Any) -> dict[str, Any]:
         return ret
     if not isinstance(param, str):
         if hasattr(param, "__metadata__"):
-            ret["description"] = param.__metadata__[0]
+            for meta in param.__metadata__:
```

Comment:
This looks like what I had about a year ago, or something. I think there's one other component (that we can handle in another PR?) to set the reserved (or desired function params) to be `is_required=False` based on this?

Reply:
good point, added logic for this scenario